serving-core/src/main/scala/com/stratio/sparta/serving/core/models/AggregationPoliciesModel.scala

```diff
@@ -91,7 +91,18 @@ object AggregationPoliciesValidator extends SpartaSerializer {
           trigger.outputs.forall(outputName => outputsNames.contains(outputName)))),
         new ErrorModel(
           ErrorModel.ValidationError_There_is_at_least_one_cube_with_triggers_with_a_bad_output,
-          "There is at least one cube with triggers that contains a bad output"))
+          "There is at least one cube with triggers that contains a bad output")),
+      (policy.cubes.forall(cube =>
+        cube.triggers.flatMap(trigger => trigger.overLast)
+          .forall(overlast => overlast.toInt % policy.sparkStreamingWindow.toInt == 0)),
+        new ErrorModel(
+          ErrorModel.ValidationError_There_is_at_least_one_cube_with_triggers_with_a_bad_output,
+          "There is at least one trigger with a bad overlast")),
+      (policy.streamTriggers.flatMap(trigger => trigger.overLast)
+        .forall(overlast => overlast.toInt % policy.sparkStreamingWindow.toInt == 0),
+        new ErrorModel(
+          ErrorModel.ValidationError_There_is_at_least_one_cube_with_triggers_with_a_bad_output,
```

Comment:
Both cases should use the variable ValidationError_There_is_at_least_one_trigger_with_a_bad_overlast right?